HTML/double-jeopardy.html

```diff
@@ -0,0 +1,163 @@
+
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+    <button class="question">
+      <div class="row-two">$200</div>
+    </button>
+
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+    <button class="question">
+      <div class="row-three">$400</div>
+    </button>
+
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+    <button class="question">
+      <div class="row-four">$600</div>
+    </button>
+
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+    <button class="question">
+      <div class="row-five">$800</div>
+    </button>
+
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+    <button class="question">
+      <div class="row-six">$1000</div>
+    </button>
+  </div>
+
+  <!-- container for Q&A boxes -->
+  <div id="container-QA-boxes">
+    <button>
+      <div class="row-seven">What is...?</div>
+    </button>
+
+    <button>
+      <div class="row-eight">Guess</div>
+    </button>
+    <button>
+      <div class="row-eight">Pass</div>
+    </button>
+  </div>
+
+  <!-- Link to JS -->
+  <scripts src="index.js"></scripts>
```

Comment:
This link is looking in the wrong location for your JS file. If you pull your HTML files out to the root level of your project you can set the `src` equal to `"./scripts/index,js"` to link it